src/standard/utils.html

```diff
@@ -306,6 +306,17 @@
         }
       }
       return elt;
+    },
+
+    /**
+     * Checks whether an element is in this element's shadow DOM tree.
+     *
+     * @method isShadowDomChild
+     * @param {HTMLElement=} node The element to be checked.
+     * @return {Boolean} true if node is in this element's shadow DOM tree.
+     */
+    isShadowDomChild: function(node) {
+      return Polymer.dom(this).getOwnerRoot() !== Polymer.dom(node).getOwnerRoot();
```

Comment:
Talked to @sorvell, the negation of this isn't "light descendant", so going back to `isLightDescendant", which is the one we really care about.